src/dataServices/dynamoDbDataService.ts

```diff
@@ -107,6 +107,9 @@ export class DynamoDbDataService implements Persistence, BulkDataAccess {
     async vReadResource(request: vReadResourceRequest): Promise<GenericResponse> {
         this.assertValidTenancyMode(request.tenantId);
         const { resourceType, id, vid, tenantId } = request;
+        if (Number.isNaN(Number(vid))) {
```

Comment:
Why not use the same code as below? `parseInt(vid, 10)`
```
const version = `parseInt(vid, 10)`;
if (!version).... 
```

Reply:
makes sense, I updated this logic